test/common/dashboards_exist.go

```diff
@@ -131,33 +121,20 @@ func TestIntegreatlyCustomerDashboardsExist(t *testing.T, ctx *TestingContext) {
 	}
 
 	expectedDashboards := getExpectedCustomerDashboard(rhmi.Spec.Type)
-	verifyExpectedDashboards(t, expectedDashboards, grafanaApiCallOutput)
+	verifyExpectedDashboards(t, expectedDashboards, removeNamespaceDashboardFolder(grafanaApiCallOutput))
 }
 
 func TestIntegreatlyMiddelewareDashboardsExist(t *testing.T, ctx *TestingContext) {
-	pods := &corev1.PodList{}
-	opts := []k8sclient.ListOption{
-		k8sclient.InNamespace(MonitoringOperatorNamespace),
-		k8sclient.MatchingLabels{"app": "grafana"},
-	}
-
 	// get console master url
 	rhmi, err := GetRHMI(ctx.Client, true)
 	if err != nil {
 		t.Fatalf("error getting RHMI CR: %v", err)
 	}
 
-	err = ctx.Client.List(goctx.TODO(), pods, opts...)
-	if err != nil {
-		t.Fatal("failed to list pods", err)
-	}
-
-	if len(pods.Items) != 1 {
-		t.Fatal("grafana pod not found")
-	}
+	monitoringGrafanaPods := getGrafanaPods(t, ctx, MonitoringOperatorNamespace)
 
 	output, err := execToPod("curl localhost:3000/api/search",
```

Comment:
Does this work? I got errors saying curl didn't exist when I tried this command

Reply:
:thinking: the functional test itself passed locally, but yeah when the curl command is run in the grafana pod terminal manually, there is an error, not sure whats happening here :man_shrugging: 

Maybe the solution here would be to get any pod **but** the grafana pod in the monitoring namespace and curl to the grafana pod ip instead ? What do you think @MStokluska ?

Reply:
@philbrookes @KevFan monitoring grafana pods do have curl, customer monitoring grafana pods don't. 
This is why I am using monitoring grafana pods to curl the customer monitoring pods. Alternative to it would be to either create a throwaway pod to connect to customer grafana or, install curl on customer grafana which was not possible due to pod security blocking it.

Reply:
Ah I see, makes sense now :+1: